launcher/electron/app/templates/change-server-dialog.html

```diff
@@ -0,0 +1,72 @@
+<!-- 
+  Copyright 2015 TWO SIGMA OPEN SOURCE, LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ -->
+
+<html>
+  <head>
+    <title>Change Beaker Server</title>
+    <script type=text/javascript>
+      var ipc = require('ipc');
```

Comment:
What do you mean? I just made it so that if the modal dialog is re-opened, then if it already exists its window is simply focused. Otherwise a new one is created.